src/node/peerman_args.cpp

```diff
@@ -1,3 +1,7 @@
+// Copyright (c) 2023-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
Is there any reason for not updating these as well?
```suggestion
// file COPYING or https://opensource.org/license/MIT.
```